src/google_takeout_metadata/processor_batch.py

```
@@ -45,44 +44,130 @@ def process_batch(batch: List[Tuple[Path, Path, List[str]]], clean_sidecars: boo
 
     except FileNotFoundError as exc:
-        raise RuntimeError("exiftool not found") from exc
+        raise RuntimeError("exiftool introuvable") from exc
     except subprocess.CalledProcessError as exc:
-        logger.error(f"exiftool failed for batch. Stderr: {exc.stderr or ''}. Stdout: {exc.stdout or ''}")
+        stderr_msg = exc.stderr or ""
+        stdout_msg = exc.stdout or ""
+        
+        # Analyser le type d'erreur pour donner un message plus clair
+        if "files failed condition" in stderr_msg or "files failed condition" in stdout_msg:
+            logger.info(f"ℹ️ Lot traité avec conditions non remplies (normal en mode append-only). "
```

Ruff (0.12.2)

String contains ambiguous `ℹ` (INFORMATION SOURCE). Did you mean `i` (LATIN SMALL LETTER I)?

(RUF001)